File: service/SurfWebAutofac/Model/Dtos/Maps/MapTop100Dto.cs
namespace Model.Dtos.Maps;

public class MapTop100Dto
{
    /// <summary>
    ///     排行
    /// </summary>
    public  int Ranking { get; set; }

    /// <summary>
    ///     玩家id
    /// </summary>
    public  long PlayerId { get; set; }

    /// <summary>
    ///     玩家名
    /// </summary>
    public  string PlayerName { get; set; }

    /// <summary>
    ///     阶段
    /// </summary>
    public int? Stage { get; set; }

    /// <summary>
    ///     通关时间
    /// </summary>
    public float Time { get; set; }

    /// <summary>
    ///     时间差距
    /// </summary>
    public float GapTime { get; set; }

    /// <summary>
    ///     日期
    /// </summary>
    public DateTime Date { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/maps/MapTop100Dto.java
package com.surfweb.system.dto.maps;

import lombok.Data;

import java.util.Date;

@Data
public class MapTop100Dto {

    /**
     * 排行
     */
    private int ranking;

    /**
     * 玩家id
     */
    private long playerId;

    /**
     * 玩家名
     */
    private String playerName;

    /**
     * 阶段
     */
    private Integer stage;

    /**
     * 通关时间
     */
    private float time;

    /**
     * 时间差距
     */
    private float gapTime;

    /**
     * 日期
     */
    private Date date;
}
